src/dom-updates.js

```diff
@@ -1,5 +1,56 @@
 const domUpdates = {
+  displayUserName(user) {
+    const nameDisplay = document.getElementById('welcome')
+    nameDisplay.innerText = `Welcome ${user.returnFirstName()}`
+  },
 
+  displayAmountSpent(user) {
+    const amountSpentDisplay = document.getElementById('amountSpent')
+    amountSpentDisplay.innerText = `You have spent $${user.calculateAmountSpent()} traveling`
+  },
+
+  displayUserTrips(user) {
+    let todaysDate = "2020/3/28"
+    const pastTripList = document.getElementById('pastTripList')
+    const upcomingTripList = document.getElementById('upcomingTripList')
+    const ongoingTripList = document.getElementById('ongoingTripList')
+    const pendingTripList = document.getElementById('pendingTripList')
+
+    user.trips.forEach((trip, i) => {
+      if (trip.status === 'pending') {
+        pendingTripList.innerHTML +=
+          `<p>${trip.correspondingDestination.destination}</p>`
+      } else {
+        if (this.calculateDate(trip, todaysDate) === 'ongoing') {
+          ongoingTripList.innerHTML +=
+            `<p>${trip.correspondingDestination.destination}</p>`
+        } else if (this.calculateDate(trip, todaysDate) === 'upcoming') {
+          upcomingTripList.innerHTML +=
+            `<p>${trip.correspondingDestination.destination}</p>`
+        } else {
+          pastTripList.innerHTML +=
+            `<p>${trip.correspondingDestination.destination}</p>`
+        }
+      }
+    });
+  },
+
+  calculateDate(trip, todaysDate) {
```

Comment:
This is an interesting pattern. I wouldn't necessarily expect a function named `calculateDate` to return a string. Perhaps a change of name would make the intent of this function more clear.